bin/cron/sync_dropbox

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env ruby
 require_relative 'only_one'
+require_relative '../../tools/hooks/hooks_utils.rb'
```

Comment:
Can we require this after the call to `only_one_running?`? We’ve had problems in the past with some calls to `require` taking longer than 1 minute to complete. If a job runs once a minute another instance of the job might start before the `only_one_running?` method gets invoked.

Reply:
Sorry for the delay -- this got buried in my inbox 😬 Done!